web/packages/teleport/src/config.ts

```diff
@@ -279,7 +279,7 @@ const cfg = {
     databasePath: `/v1/webapi/sites/:clusterId/databases/:database`,
     databasesPath: `/v1/webapi/sites/:clusterId/databases?searchAsRoles=:searchAsRoles?&limit=:limit?&startKey=:startKey?&query=:query?&search=:search?&sort=:sort?`,
 
-    databaseServers: {
+    databaseServer: {
```

Comment:
i reverted the changes to this, b/c this just describes the resource (like `db_server`) and it contains different actions against the resource like [this](https://github.com/gravitational/teleport/blob/5b686f538281679175cdd231178c85018adb9472/web/packages/teleport/src/config.ts#L911)